bazel/BUILD.hiredis

```diff
@@ -0,0 +1,47 @@
+COPTS = [] + select({
+    "@bazel_tools//src/conditions:windows": [
+        "-D_CRT_DECLARE_NONSTDC_NAMES=0",  # don't define off_t, to avoid conflicts
+    ],
+    "//conditions:default": [
+    ],
+})
+
+cc_library(
```

Comment:
comment why this is needed? (vs. directly merge it to cc_library(name = "hiredis"...

Reply:
Oops I thought I kept the note for this, thanks for catching!